soccer/gameplay/root_play.py

```diff
@@ -192,7 +195,7 @@ def goalie_id(self):
 
     @goalie_id.setter
     def goalie_id(self, value):
-        self._goalie_id = None if value == -1 else value
+        self._goalie_id = value
```

Comment:
@jgkamat I think this function gets called through the c++ wrapper which makes it pretty hard to track. Previously, the c++ side represented the goalie id not being set with -1 while the python side used None. I have changed it so that the python side matches the c++ side which was necessary, I can't remember the exact reason but I think it made it possible for the role assignment system to handle the goalie going out of vision or something. Anyway, long story short the check is no longer necessary because python just mimics exactly what the c++ side sends it.